sbt-cross/src/main/scala/sbtcross/CrossPlugin.scala

```diff
@@ -0,0 +1,24 @@
+package sbtcross
+
+import sbt._
+import StringUtilities.nonEmpty
+
+import scala.language.implicitConversions
+
+object CrossPlugin extends AutoPlugin {
+  override def requires: Plugins = plugins.JvmPlugin
+
```

Comment:
It works already and I have a test for it:
https://github.com/MasseGuillaume/sbt-cross-project/blob/simple2/sbt-cross-test/src/sbt-test/new-api/crossDependenciesWithoutCross/build.sbt